jobs/forms.py

```diff
@@ -7,11 +7,20 @@
 
 class JobForm(forms.ModelForm):
     description = forms.CharField(widget=CKEditorWidget())
+    website = forms.URLField(
+        initial='http://',
+        help_text='Rememeber to start with http:// or https://'
+    )
+    cities = forms.CharField(
+        help_text="If you have opportunities in several \
+            countries, please either specify that in the job description, \
+            or add job opportunities for each country in which you have a space."
+        )
```

Comment:
I am a bit undecided about the help text for the cities/country.  I think we might run into problems later if we want to do searches/filters based on country.  The help test seems to say that if you have opportunities in >1 country, you could add all the cities here (London, Rome, Paris etc) although you'd only have been able to select one country, and then you just say in the description that it applies to > 1 country.  If we later wanted to search/filter by country, and we'd entered UK as the country, this opportunity wouldn't be returned from a filter on Italy or France.

Cities isn't such an issue as we have a free text field, so if we want to search on it later we're going to need to parse the text anyway.

Do you think it would be better to require users to enter separate job opportunities if they have availability in more than one country?  (Rather than giving them the option to do it either way).  If so, the help text would be something like:
"List the cities where this opportunity is available.  If you have opportunities in cities in more than one country, please submit a separate job opportunity per country."